openedx_events/learning/signals.py

```diff
@@ -136,3 +137,14 @@
         "configuration": CourseDiscussionConfigurationData
     }
 )
+
+# .. event_type: org.openedx.learning.course.persistent_grade.summary.v1
+# .. event_name: PERSISTENT_GRADE_SUMMARY_CHANGED
+# .. event_description: emitted when a persistent grade is created or updated in the course
+# .. event_data: PersistentCourseGradeData
+PERSISTENT_GRADE_SUMMARY_CHANGED = OpenEdxPublicSignal(
```

Comment:
Why is this called `PERSISTENT_GRADE_SUMMARY_CHANGED`, if it only contains `PersistentCourseGradeData`?

Reply:
It is called this way because it is a summary of the grades you have submitted in your course, this grade will be updated every time you get a new grade. do you think we should modify its name?

Reply:
I understand now. This signal/event is triggered every time this grade summary is changed by adding new `PersistentCourseGradeData`. The name is accurate.